elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/parser/handlebars/HandlebarsHelper.java

```diff
@@ -193,4 +194,23 @@ public String trimColumnReferences(String str) {
         }
         return str;
     }
+
+    /**
+     * Get Class Name for provided modelName from static models if available else default value.
+     * @param modelName model name.
+     * @return class name.
+     */
+    public String getJoinClassName(String modelName) {
+        return ElideDynamicEntityCompiler.getStaticModelClassName(modelName, EMPTY_STRING,
```

Comment:
There is a NO_VERSION constant you can use here. you can use here.

Reply:
👍